### PR TITLE
feat: rename "Xpla Wallet" to "Xpla Vault"

### PR DESCRIPTION
**Pull request by caramis**

### CODE REVIEW

File: content/docs/develop/guides/smart-contract-guide/interacting-with-the-contract.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ title: Interacting with the Contract
 
 {{< hint info >}}
 **Note**  
-You can also follow these steps in the official web wallet for Xpla, [Xpla wallet](https://wallet.xpla.io).
+You can also follow these steps in the official web wallet for Xpla, [Xpla Vault](https://wallet.xpla.io).
```
Comment (jhlee-young) [inline content/docs/develop/guides/smart-contract-guide/interacting-with-the-contract.md]: the link also should be fixed

File: content/docs/develop/tools/wallet-provider/mobile-wallet.md [hunk 2]
```diff
@@ -5,17 +5,17 @@ title: Mobile Wallet
...
 - Create wallets and send tokens
 - Get involved with staking by browsing through validator information and delegating XPLA tokens.
 - Use QRCodes for easy interactions when sending assets and recovering wallets
 
 ## URL Scheme
 
-Xpla mobile wallet includes a custom [URL Scheme](https://developer.apple.com/documentation/xcode/defining-a-custom-url-scheme-for-your-app) that lets developers trigger different actions in the app.
+Xpla mobile vault includes a custom [URL Scheme](https://developer.apple.com/documentation/xcode/defining-a-custom-url-scheme-for-your-app) that lets developers trigger different actions in the app.
 
 These URL handlers can be opened by scanning a QR code or opening the link directly.
```
Comment (jhlee-young) [inline content/docs/develop/tools/wallet-provider/mobile-wallet.md:21]: could you consider whether the `URL` below(line 29) need to be fixed

File: content/docs/full-node/manage-a-validator/validator-faq.md [hunk 3]
```diff
@@ -123,13 +123,13 @@ A validator operator's "self-bond" refers to the amount of XPLA delegated to its
 
 ### Can I Delegate to a Validator Outside of the Active Set?
 
-You can still delegate to a validator even if they do not appear on Xpla wallet. Simply add the xplavaloper address of your desired validator to the end of the following URL:
+You can still delegate to a validator even if they do not appear on Xpla Vault. Simply add the xplavaloper address of your desired validator to the end of the following URL:
 
 `https://wallet.xpla.io/validator/<xplavaloper-address>`
```
Comment (jhlee-young) [inline content/docs/full-node/manage-a-validator/validator-faq.md]: this link also needs to be updated

File: content/docs/learn/xpla-vault/download/mobile-vault.md [hunk 4]
```diff
@@ -1,24 +1,24 @@
...
 
-## Download the Xpla Wallet App
+## Download the Xpla Mobile Vault
 
 1. Tap the download link for your operating system. Your device’s app store will open.
 
-   - [Xpla wallet for iOS](https://apps.apple.com/app/...)
-   - [Xpla wallet for Android](https://play.google.com/store/apps/details?id=io.xpla.wallet)
+   - [Xpla Vault for iOS](https://apps.apple.com/app/...)
+   - [Xpla Vault for Android](https://play.google.com/store/apps/details?id=io.xpla.wallet)
```
Comment (jhlee-young) [inline content/docs/learn/xpla-vault/download/mobile-vault.md]: `id=io.xpla.vault`?

**Comment by caramis** [comment]
Fixed ! @jhlee-young @psy2848048

**Review by psy2848048 — APPROVED** [review]

**Review by jhlee-young — APPROVED** [review]
LGTM 👍

**Pull request merged**